platform/ui/src/components/AllInOneMenu/Item.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactNode, useCallback, useContext } from 'react';
+import { MenuContext } from './Menu';
+
+type ItemProps = {
+  label: string;
+  secondaryLabel?: string;
+  icon?: ReactNode;
+  onClick?: () => void;
+};
+
+const Item = ({ label, secondaryLabel, icon, onClick }: ItemProps) => {
+  const { hideMenu } = useContext(MenuContext);
+
+  const onClickHandler = useCallback(() => {
+    hideMenu();
+    onClick?.();
+  }, [hideMenu, onClick]);
+
+  return (
+    <div
+      className="all-in-one-menu-item all-in-one-menu-item-effects"
+      onClick={onClickHandler}
+    >
+      {icon && (
+        <>
+          {icon}
+          <div className="w-2"></div>
```

Comment:
isn't this like padding left/right or something? it is odd to have empty div of w-2